okhttp/src/main/java/com/squareup/okhttp/ConnectionSpec.java

```diff
@@ -78,6 +78,12 @@
   /** Used if tls == true. The TLS protocol versions to use. */
   private final String[] tlsVersions;
 
+  /**
+   * Used if tls == true. Whether the first tlsVersion must be enabled for
+   * {@link #isCompatible(javax.net.ssl.SSLSocket)} to return true.
+   */
+  private final boolean requireFirstTlsVersion;
```

Comment:
Hmmm . . . I think I see that it can't be, because then we might skip MODERN_TLS entirely just because TLS_1_2 is absent.